ReactiveObjC/UISearchBar+RACSignalSupport.m

```diff
@@ -0,0 +1,48 @@
+//
+// Created by Igor Vasilenko on 09/09/16.
+// Copyright (c) 2016 GitHub. All rights reserved.
+//
+
+#import "UISearchBar+RACSignalSupport.h"
+
+#import <ReactiveObjC/EXTScope.h>
+#import "NSObject+RACDeallocating.h"
+#import "NSObject+RACDescription.h"
+#import "RACDelegateProxy.h"
+#import "RACSignal+Operations.h"
+#import <objc/runtime.h>
+
+@implementation UISearchBar (RACSignalSupport)
+
+static void RACUseDelegateProxy(UISearchBar *self) {
+    if (self.delegate == self.rac_delegateProxy) return;
+
+    self.rac_delegateProxy.rac_proxiedDelegate = self.delegate;
+    self.delegate = (id)self.rac_delegateProxy;
```

Comment:
The indentation is this file should be tabs, not spaces, to match the project conventions.